issue-diff.py

```diff
@@ -26,26 +26,49 @@
 # unicode character in its summary.
 BUG_SUMMARY = 'Missing issue references from {project}/{package} in {factory}/{package}'
 BUG_TEMPLATE = u'{message_start}\n\n{issues}'
-MESSAGE_START = 'The following issues were referenced in the changelog for {project}/{package}, but where not found in {factory}/{package} after {newest} days. Review the issues and submit changes to {factory} as necessary.'
+MESSAGE_START = 'The following issues were referenced in the changelog for {project}/{package}, but where not found in {factory}/{package} after {newest} days. Review the issues and submit changes to {factory} to ensure all relevant changes end up in {factory} which is used as the basis for the next SLE version. For more information and details on how to go about submitting the changes see https://mailman.suse.de/mlarch/SuSE/research/2017/research.2017.02/msg00051.html.'
 ISSUE_SUMMARY = u'[{label}]({url}) owned by {owner}: {summary}'
 ISSUE_SUMMARY_PLAIN = u'[{label}]({url})'
 
 
-def bug_create(bugzilla_api, meta, cc, summary, description):
+def bug_create(bugzilla_api, meta, assigned_to, cc, summary, description):
     createinfo = bugzilla_api.build_createbug(
         product=meta[0],
         component=meta[1],
         version=meta[2],
         severity='normal',
         op_sys='Linux',
         platform='PC',
+        assigned_to=assigned_to,
         cc=cc,
         summary=summary,
         description=description)
     newbug = bugzilla_api.createbug(createinfo)
 
     return newbug.id
 
+def user_email(apiurl, userid):
+    url = osc.core.makeurl(apiurl, ('person', userid))
+    root = ET.parse(osc.core.http_GET(url)).getroot()
+    email = root.find('email')
+    return email.text if email is not None else None
+
+def bug_owner(apiurl, package):
+    query = {
+        'binary': package,
+    }
+    url = osc.core.makeurl(apiurl, ('search', 'owner'), query=query)
```

Comment:
It is, `apiurl_default` is OBS and only used for Factory package list calls. `apiurl` is always suse.de (or to whatever the `-A` option is set)